### ISSUE TITLE
Generateur de configuration HS

### ISSUE THREAD

**Issue by peytonjol** [c0mm0n/edgemax4orange#5]
http://www.homelabs.fr/ubiquiti-edgemax/generateur-de-configuration/

Affiche : Erreur lors de la connexion à la base de données

Merci

**Comment by c0mm0n** [comment]
C'est de retour.

**Comment by TuXbOy06** [comment]
Hello fonctionne pas mieux
Quand on clique sur generer rien se passe.
 merci

**Comment by soleshoe** [comment]
Yep, toujours HS, aucune réaction au clic sur le bouton 'générer'

Thx
